jira_agile_metrics/calculators/waste_test.py

```diff
@@ -196,7 +192,4 @@ def test_different_done_column(jira, settings):
 
     assert data.to_dict('records') == [
         {'key': 'A-2', 'last_status': 'Committed', 'resolution': 'Withdrawn', 'withdrawn_date': Timestamp('2018-01-07 02:02:02')},
-        {'key': 'A-3', 'last_status': 'Done',      'resolution': 'Withdrawn', 'withdrawn_date': Timestamp('2018-01-08 02:02:02')},
-        {'key': 'A-6', 'last_status': 'foobar',    'resolution': 'Withdrawn', 'withdrawn_date': Timestamp('2018-01-06 02:02:02')},
-        {'key': 'A-7', 'last_status': None,        'resolution': 'Withdrawn', 'withdrawn_date': Timestamp('2018-01-06 02:02:02')},
     ]
```

Comment:
... and here

Reply:
As described in https://github.com/DeloitteDigitalUK/jira-agile-metrics/pull/27#issuecomment-636308642 I think that `waste.py` should only consider `active_columns` starting from `committed_column` up to and excluding `done_column`. Columns before `committed_columns` as well as after and including `done_column` should be ignored.